.github/workflows/container-image.yaml

```diff
@@ -1,53 +1,69 @@
-          TAG="latest"
-        else
-          TAG=$BRANCH
-        fi
-        docker login docker.pkg.github.com -u $USER -p ${{ secrets.REGISTRY_TOKEN }}
-        docker build -t aws-ebs-csi-driver . --target amazonlinux
-        docker tag aws-ebs-csi-driver docker.pkg.github.com/$GITHUB_REPOSITORY/$IMAGE:$TAG
-        docker push docker.pkg.github.com/$GITHUB_REPOSITORY/$IMAGE:$TAG
-    - name: Push to Dockerhub registry
-      run: |
-        BRANCH=$(echo $GITHUB_REF | cut -d'/' -f3)
-        REPO=amazon/aws-ebs-csi-driver
-        if [ "$BRANCH" = "master" ]; then
-          TAG="latest"
-        else
-          TAG=$BRANCH
-        fi
-        docker login -u ${{ secrets.DOCKERHUB_USER }} -p ${{ secrets.DOCKERHUB_TOKEN }}
-        docker buildx build \
-          -t $REPO:$TAG \
-          --platform=linux/arm64,linux/amd64 \
-          --output="type=image,push=true" . \
-          --target=amazonlinux
-
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build debian target
+        run: |
+          docker buildx build \
+            -t $DEB_BUILD_TAG \
+            --platform=linux/arm64,linux/amd64 \
+            --output="type=image,push=false" .
+      - name: Build amazonlinux target
+        run: |
+          docker buildx build \
+            -t $AL2_BUILD_TAG \
+            --platform=linux/arm64,linux/amd64 \
+            --output="type=image,push=false" . \
+            --target=amazonlinux
+      - name: Set environment variables
+        run: |
+          BRANCH=$(echo $GITHUB_REF | cut -d'/' -f3)
+          echo "BRANCH=$BRANCH" >> $GITHUB_ENV
+          if [ "$BRANCH" = "master" ]; then
+            TAG="latest"
+          else
+            TAG=$BRANCH
+          fi
+          echo "TAG=$TAG" >> $GITHUB_ENV
+      - name: Login to GitHub
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.REGISTRY_TOKEN }}
+      - name: Push debian target to GitHub
+        run: |
+          DEB_PUSH_TAG="docker.pkg.github.com/$GITHUB_REPOSITORY/$IMAGE:$TAG"
+          docker tag $DEB_BUILD_TAG $DEB_PUSH_TAG
+          docker push $DEB_PUSH_TAG
+      - name: Push amazonlinux target to GitHub
+        run: |
+          AL2_PUSH_TAG="docker.pkg.github.com/$GITHUB_REPOSITORY/$IMAGE:$TAG-amazonlinux"
+          docker tag $AL2_BUILD_TAG $AL2_PUSH_TAG
+          docker push $AL2_PUSH_TAG
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Push amazonlinux target to Docker Hub
+        run: |
+          AL2_PUSH_TAG="amazon/$IMAGE:$TAG-amazonlinux"
+          docker tag $AL2_BUILD_TAG $AL2_PUSH_TAG
+          docker push $AL2_PUSH_TAG
```

Comment:
I realized I changed this from docker buildx build --push=true, so this won't push the arm64 manifest.... need to fix.